src/Raven.Client/Documents/Operations/ETL/Elasticsearch/ElasticsearchConnectionString.cs

```diff
@@ -83,10 +83,11 @@ public override DynamicJsonValue ToJson()
 
         public override DynamicJsonValue ToAuditJson()
         {
-            DynamicJsonValue json = base.ToAuditJson();
+            DynamicJsonValue json = base.ToJson();
```

Comment:
Shouldn't be `base.ToAuditJson();`?

Reply:
i've changed it because base ToAuditJson() calls ToJson() overridde on the object, including the credentials in the process

Reply:
refactored base ToAuditJson() to not call ToJson override